instrumentation/micrometer/micrometer-1.5/library/src/main/java/io/opentelemetry/instrumentation/micrometer/v1_5/OpenTelemetryMeterRegistryBuilder.java

```diff
@@ -16,6 +18,9 @@
 
   private final OpenTelemetry openTelemetry;
   private Clock clock = Clock.SYSTEM;
+  private TimeUnit baseTimeUnit =
+      TimeUnitHelper.parseConfigValue(
+          Config.get().getString("otel.instrumentation.micrometer.base-time-unit"));
```

Comment:
can you add this to the README.md similar to #4981?

Reply:
Good point, added.